bgpd/bgp_mplsvpn.c

```diff
@@ -4074,6 +4074,32 @@ void bgp_vpn_leak_export(struct bgp *from_bgp)
 	}
 }
 
+void bgp_vpn_release_label(struct bgp *bgp, afi_t afi, bool reset)
```

Comment:
please add a comment block explaining the dual uses of this new function

Reply:
Added the comment